homeassistant/components/sharkiq/__init__.py

```diff
@@ -41,6 +48,18 @@ async def async_connect_or_timeout(ayla_api: AylaApi) -> bool:
     return True
 
 
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Migrate version 1 to version 2."""
+    if config_entry.version == 1:
+        LOGGER.info("Migrating from version %s", config_entry.version)
+        new = {**config_entry.data}
+        new[CONF_REGION] = SHARKIQ_REGION_DEFAULT
+
+        config_entry.version = 2
+        hass.config_entries.async_update_entry(config_entry, data=new)
+    return True
```

Comment:
I'm sorry, I know @mib1185 gave the advise in the previous review, but it is actually not correct / wished for.

Migrations are preventing downgrades, as it is no longer backward compatible. Cases like these can just be handled with a default, or, update the config entry on integration init. 

There is no need to make a backward incompatible change in this case.

Reply:
oh sorry, my bad ... was not aware of this intention of the migration.